schemas/participant.json

```diff
@@ -0,0 +1,241 @@
+{
+    "name": "participant",
+    "description": "The collection of data elements related to a specific donor in a PCGL program.",
+    "meta": {
+      "parent": "study"
+    },
+    "fields": [
+      {
+        "name": "study_id",
+        "valueType": "string",
+        "description": "Unique identifier of the study.",
+        "meta": {
+          "primaryId": true,
+          "examples": "TEST-CA",
+          "foreignKey": "study.study_id",
+          "notes": "Source:MOH/ARGO",
+          "displayName": "Study ID",
+          "source": "MOH/ARGO"
+        },
+        "restrictions": {
+          "required": true
+        }
+      },
+      {
+        "name": "submitter_participant_id",
+        "valueType": "string",
+        "description": "Unique identifier for the participant, assigned by the data provider.",
+        "meta": {
+          "primaryId": true,
+          "examples": "90234,BLD_donor_89,AML-90",
+          "displayName": "Submitter Participant ID",
+          "source": "MOH/ARGO",
```

Comment:
As per jon's suggestion:
https://github.com/Pan-Canadian-Genome-Library/data-dictionary/pull/1#discussion_r1541271035

Source and mappings were added to meta